content/en/apps/payments/payments-sdk.md

```diff
@@ -0,0 +1,124 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: Corda Payments SDK
+title: Send and receive payments with the demo CorDapps
+weight: 200
+---
+
+With the Corda Payments SDK you can build and deploy payment-enabled CorDapps on your network. Use this documentation to add all the required Payment SDK dependencies to an existing CorDapp. Once the dependencies have been added to your CorDapp, it can be configured to initiate payments, via the network's Payments Agent.
```

Comment:
```suggestion
With the Corda Payments SDK you can build and deploy payment-enabled CorDapps on your network. Use this documentation to add all the required Payments SDK dependencies to an existing CorDapp. Once the dependencies have been added to your CorDapp, it can be configured to initiate payments, via the network's Payments Agent.
```